### ISSUE TITLE
Normalize/test the save for keras, pytorch in mlmodels/utils.py

### ISSUE THREAD

**Issue by arita37** [arita37/mlmodels#51]
This part of the code : 

https://github.com/arita37/mlmodels/blob/48ad2f28c1905c7fce5b626b9bc424e8400046ef/mlmodels/util.py#L393
